packages/ecs-agent/ecs-agent.spec

```diff
@@ -51,12 +38,6 @@ Source200: etc-systemd-system-ecs.service.d.mount
 # Ecs logdog configuration
 Source300: logdog.ecs.conf
 
-# Patches are numbered according to which source they apply to
-# Patches 0000 - 0999 apply to Source0
-# Patches 1000 - 1999 apply to Source1
-# Patches 2000 - 2999 apply to Source2
-# See the %prep section for the implementation of this logic
-
 # Bottlerocket-specific - filesystem location of the pause image
 Patch0001: 0001-bottlerocket-default-filesystem-locations.patch
```

Comment:
I am probably confused, but GitHub tells me that this file was renamed out of this directory and into amazon-ecs-cni-plugins (and the spec file there incorporates it). Does this patch file appear in both packages?

Reply:
Oh! I _am_ confused. The very-similar-sounding patch 1001-bottlerocket-default-filesystem-locations.patch moved.